packages/worker/src/worker-options.ts

```diff
@@ -337,7 +353,7 @@ export type WorkerOptionsWithDefaults = WorkerOptions &
      *
      * This value is not exposed at the moment.
      *
-     * @default 8
+     * @default 1 for reuseV8Context, otherwise 8
```

Comment:
Is `workflowThreadPoolSize > 1` valid with `reuseV8Context = true`? Might it be pertinent to set default to something else than 1 for `reuseV8Context = true`?

Reply:
There's not much value in it AFAICT.
It's better to create less threads until we see a need for more, it'll make startup faster.